recipes/rhapsody/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set version = "0.2.1" %}
+{% set name = "rhapsody" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/r/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d73e0af9f57fdb7e48bd7bab4f946fbeb0240f6d44cb47e0257a277df7051beb
+
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+  entry_points:
+    - qiime2.plugins = rhapsody.q2.plugin_setup:plugin
+
+requirements:
+  host:
+    - python <3.7
+    - pip
+    - numpy >=1.13.1
+
+  run:
+    - python <3.7
+    - ipython
+    - pandas
+    - click
+    - tensorflow >1.7
+    - numpy >1.13
+    - scikit-bio >0.5.4
+    - nose >=1.3.7
+    - seaborn
+    - patsy
+    - scikit-learn
+    - tqdm
+    - h5py
+    - biom-format
+
+test:
+  imports:
+    - rhapsody
+
+about:
+  home: https://github.com/biocore/rhapsody
+  license: BSD 3-Clause
```

It's a little thing but for consistency with the SPDX scheme:
```suggestion
  license: BSD-3-Clause
```